> Solving the next problem at: 2024-09-27 08:32:44 UTC
> Search algorithm: bfs
> Instance ID=pegsol-08-strips/pp04-err-rate-0-5
> Plan length=13
> Domain class: pegsol-08-strips
> Instance name: pp04-err-rate-0-5
> Ground plan:
['(jump-new-move pos-0-2 pos-0-3 pos-0-4)', '(end-move pos-0-4)', '(jump-new-move pos-1-2 pos-1-3 pos-1-4)', '(end-move pos-1-4)', '(jump-new-move pos-0-4 pos-1-4 pos-2-4)', '(end-move pos-2-4)', '(jump-new-move pos-5-2 pos-5-3 pos-5-4)', '(end-move pos-5-4)', '(jump-new-move pos-2-4 pos-3-4 pos-4-4)', '(end-move pos-4-4)', '(jump-new-move pos-5-4 pos-4-4 pos-3-4)', '(end-move pos-3-4)', '(jump-new-move pos-3-5 pos-3-4 pos-3-3)']
> Lifted plan:
['jump-new-move', 'end-move', 'jump-new-move', 'end-move', 'jump-new-move', 'end-move', 'jump-new-move', 'end-move', 'jump-new-move', 'end-move', 'jump-new-move', 'end-move', 'jump-new-move']
>>>  Vanilla ground repair length:
2

>>>  Vanilla ground repair:
jump-new-move,(in-line ?over ?over ?to),precPos,-1
jump-new-move,(in-line ?to ?to ?to),effPos,1

An error occurred during A* search: name 'logger' is not defined
Traceback (most recent call last):
  File "/home/nader/projects/lifted-white-plan-domain-repair/instance_solver.py", line 70, in solve_instance
    path, goal_node = searcher.find_path(log_interval=log_interval)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nader/projects/lifted-white-plan-domain-repair/search_partial_grounding/search_algorithms.py", line 49, in find_path
    for neighbor in current_node.get_neighbors():
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nader/projects/lifted-white-plan-domain-repair/search_partial_grounding/node.py", line 167, in get_neighbors
    logger.debug(f'> Current state:\n{self.current_state}')
    ^^^^^^
NameError: name 'logger' is not defined. Did you mean: 'self.logger'?

> Time spent on this problem: 0.0 hours = 0.0 minutes = 0.19 seconds
> Subprocess finished.
